### PR TITLE
Change of runner selection

### PR DESCRIPTION
**Pull request by Tatarinho**

### CODE REVIEW

**Review by alicja-gruzdz — APPROVED** [review]

**Review by MichalCz — APPROVED** [review]

**Review by jan-warchol — APPROVED** [review]
Looks good, only two minor things.

File: packages/adapters/src/process-sequence-adapter.ts [hunk 1]
```diff
@@ -27,9 +27,11 @@ async function getRunnerConfigForStoredSequence(sequencesRoot: string, id: strin
     const packageJson = await readStreamedJSON(createReadStream(packageJsonPath));
 
     const validPackageJson = await sequencePackageJSONDecoder.decodeToPromise(packageJson);
+    const engines = validPackageJson.engines ? { ...validPackageJson.engines } : {};
```
[the remark this answers is not in the document]
Comment (Tatarinho) [reply packages/adapters/src/process-sequence-adapter.ts:30]: here i copy pasted exactly same behaviour as we have for docker-sequence-adapter, i just assumed its safe and tested

**Review by MichalCz — APPROVED** [review]

File: packages/adapters/src/process-instance-adapter.ts [hunk 2]
```diff
@@ -49,12 +49,20 @@ IComponent {
...
-            return [
-                "python3",
-                path.resolve(__dirname, "../../../python/runner/runner.py")
-            ];
+            if (engines.length > 1) {
+                throw new Error("Incorrect config pased to SequenceConfig," +
+                "'engines' field can't contain more than one element");
+            }
+
+            if (engines[0] === "python3") {
```
Comment (patuwwy) [inline packages/adapters/src/process-instance-adapter.ts:59]: ```if ("python3" in config.engines) {````
or 
```if (~engines.indexof("python3") {```
of 
```if (engines.includes("pyhon3") {```

coz, "python3" could be on any place in keys if there is more than 1 required engine specified.

**Review by patuwwy — CHANGES_REQUESTED** [review]
one comment from me :)

File: packages/adapters/src/process-instance-adapter.ts (hunk 2 above)
Comment (Tatarinho) [reply packages/adapters/src/process-instance-adapter.ts]: if (engines.length > 1)  solve this assumption for now, but I've changed as you requested, it's more safe solution for future :)

File: packages/adapters/src/process-instance-adapter.ts [hunk 3]
```diff
@@ -48,9 +48,15 @@ IComponent {
     }
 
     getRunnerCmd(config: SequenceConfig) {
-        if (config.entrypointPath.endsWith(".py")) {
-            this.logger.trace(gotPython);
+        const engines = Object.keys(config.engines);
+
+        if (engines.length > 1) {
+            throw new Error("Incorrect config pased to SequenceConfig," +
```
Comment (patuwwy) [inline packages/adapters/src/process-instance-adapter.ts:54]:
```suggestion
            throw new Error("Incorrect config passed to SequenceConfig," +
```

**Review by patuwwy — APPROVED** [review]

**Pull request merged**